app/uk/gov/hmrc/thirdpartyapplication/models/UserId.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.thirdpartyapplication.models
+
+import java.util.UUID
+
+case class UserId(value: String) extends AnyVal
```

Comment:
Should UserId be a string or a UUID?

Reply:
We can change this afterwards when we're also ready to apply the APP_ID PR

Reply:
UUID probably. However it is consistent with how ApplicationId currently works.